webapp/angular.json

```
@@ -93,7 +93,11 @@
             "polyfills": "src/ts/polyfills.ts",
             "tsConfig": "tsconfig.spec.json",
             "karmaConfig": "tests/karma/karma-unit.conf.js",
-            "progress": true,
+            "codeCoverage": true,
+            "codeCoverageExclude": [
+              "src/js/bootstrapper/**",
+              "src/js/enketo/**"
+            ],
```

I know this is a draft PR, but while looking at the report I noticed a couple other exclusion possibilities so I figured I'd note them here.

```suggestion
            "codeCoverageExclude": [
              "src/js/bootstrapper/**",
              "src/js/enketo/**",
              "ts/environments/**",
              "ts/polyfills.ts"
            ],
```

Ideally the environment stuff doesn't get any branches anyway, but even if it does (yikes) I don't know how we'd write tests for the "production" sections.

---

oh, yea, definitely. we can expand this list upon discussing. 

I _know_ we test bootstrapper with mocha (https://github.com/medic/cht-core/tree/master/webapp/tests/mocha/unit) and I _know_ we have 2 mocha tests for enketo :D https://github.com/medic/cht-core/tree/master/webapp/tests/mocha/unit/enketo

---

I've added those 2 you indicated, but haven't really found others (we can complete this list as exclusions become obvious).